src/pages/patch/patchTabs/Util.tsx

```diff
@@ -0,0 +1,62 @@
+import { SorterResult } from "antd/es/table/interface";
+import {
+  TaskResult,
+  SortDirection,
+  SortOrder,
+  TaskSortCategory,
+} from "gql/generated/types";
+
+// takes sort input from the antd table and translates into part of the query string
+export const toSortString = (
+  sorts: SorterResult<TaskResult> | SorterResult<TaskResult>[]
+) => {
+  let sortStrings: string[] = [];
+  const shortenSortOrder = (order: string) =>
+    order === "ascend" ? SortDirection.Asc : SortDirection.Desc;
+  if (Array.isArray(sorts)) {
+    sorts.forEach((sort) => {
+      const singleSortString = `${sort.columnKey},${shortenSortOrder(
+        sort.order
+      )}`;
+      sortStrings = sortStrings.concat(singleSortString);
+    });
+  } else {
+    sortStrings = sortStrings.concat(
+      `${sorts.columnKey},${shortenSortOrder(sorts.order)}`
+    );
+  }
+
+  return sortStrings.join(";");
+};
+
+// takes a sort query string and parses it into valid GQL params
+export const parseSortString = (sortQuery: string | string[]): SortOrder[] => {
```

Comment:
In what case would this take in an array of sortStrings?

Reply:
I'm not sure when we would actually get an array, but this is the type returned by queryString.parse which I pass directly into this